serializer/serializer.c

```diff
@@ -312,6 +316,13 @@ status_t ta_send_transfer_req_deserialize(const char* const obj,
   if (json_result != NULL && json_result->valuestring != NULL) {
     tag_len = strnlen(json_result->valuestring, NUM_TRYTES_TAG);
 
+    for (int i = 0; i < tag_len; i++) {
+      if (json_result->valuestring[i] & (unsigned)128) {
+        ret = SC_SERIALIZER_JSON_PARSE_UNICODE;
```

Comment:
`SC_SERIALIZER_JSON_PARSE_UNICODE` seems weird as we don't support it yet. Let's rename to ascii related.

Reply:
maybe `SC_SERIALIZER_JSON_PARSE_NONASCII` ?

Reply:
Just `SC_SERIALIZER_JSON_PARSE_ASCII`. It means error when parsing ascii or some sort.